### PR TITLE
Change the pronto to rubocop

### PR DESCRIPTION
**Pull request by xiangge**
1. Change the pronto to rubocop to reduce the times of calling github apis and forbid the 403 errors in travis.
2. fix the format issues.

Even we add a PRONTO_GITHUB_ACCESS_TOKEN, when we create pr from a fork, there should occur errors because the Env Vars doesn't work for forks, https://docs.travis-ci.com/user/pull-requests/#Pull-Requests-and-Security-Restrictions , so close pr #48, and create this patch.

And the flay doesn't work when calling the pronto, so here I just add a rubocop command.

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -38,4 +38,4 @@
 end
 
 # TODO: decide if this is okay to do
-include Fixtures
+# include Fixtures
```
Comment (simozhan) [inline spec/spec_helper.rb]: If we have changed to use the module namespace  in other files, maybe we can remove these two lines.

File: .travis.yml [hunk 2]
```diff
@@ -13,16 +13,15 @@ install:
 - git config --global user.email "pdc@product-definition-center.com"
 script:
 - bundle exec rake test
-- >
+- |
```
Comment (simozhan) [inline .travis.yml]: maybe this is change is not necessary?

Comment (xiangge) [reply .travis.yml]: Yes, rollback the codes now.

File: spec/spec_helper.rb (hunk 1 above)
Comment (xiangge) [reply spec/spec_helper.rb]: Removed, thx.

**Pull request merged**